community/DOCS-CONTRIBUTING.md

```diff
@@ -1,5 +1,9 @@
 # Contributing to the Knative Documentation
 
+- [Before you begin](#before-you-begin)
+- [Contributing to the documentation](#contributing-to-the-documentation)
+- [Docs contributor roles](#docs-contributor-roles)
```

Comment:
Thanks @samodell , but there is still no process for how to become a Member if the contributor can met the requirement for applying as a Member. For k8s, we have a repo https://github.com/kubernetes/org and the people who want to be a member can create an issue and will be added as a Member if approved. How can I apply for Knative?

Reply:
Right now the process is that your nominator sends an email to the Steering Committee to have you added to the Knative org -- I did that yesterday, and got confirmation that you were added. You should start seeing the benefits of being a member, such as not needing the "ok-to-test" flag on your PRs, and ability to assign people.

Reply:
Ah, and I can see you now have the Knative badge on your GitHub profile and you show up in the list of members :) https://github.com/orgs/knative/people?utf8=%E2%9C%93&query=gyliu513

Reply:
Yes, thanks @samodell , can you please add some detail for the process in the document at somewhere for the process so that others who want to join knative org can follow the process as well? :-)

Reply:
Ah, fair point. I'll state the steps explicitly.